src/mm/dbcsr_acc_operations.F

```diff
@@ -226,9 +170,9 @@ SUBROUTINE dbcsr_acc_transpose(trs_stack, offset, nblks, datatype, buffer, m, n,
       IF (careful_mod) CALL timeset(routineN, error_handle)
       istat = 0
 
-      ! Limits the call to small kernels
-      IF (m .LE. max_kernel_dim .AND. &
-          n .LE. max_kernel_dim) THEN
+      ! Call batched in-place transpose in libsmm_acc
+      IF (m .LE. 80 .AND. &
+          n .LE. 80) THEN
```

Comment:
here we are losing documentation and introduce a magic constant, maybe introduce a `parameter` for that and document it?

Reply:
Good catch, I'll fix this ...

Reply:
fixed in commit 7a90508